resources/bundles/org.eclipse.core.resources/schema/refreshProviders.exsd

```diff
@@ -64,6 +69,13 @@
                </appInfo>
             </annotation>
          </attribute>
+         <attribute name="priority" type="string">
```

Comment:
Not sure about type, if possible, integer would be better.
In any case please give some hints in the documentation about allowed range, e.g is "-1" or `Long.MAX_LONG` are OK or not etc.

Reply:
Only string, class or boolean are allowed :-(

Reply:
I have now clarified the description.